queries.sql

```diff
@@ -1,3 +1,25 @@
-/*Queries that provide answers to the questions from all projects.*/
+/*Queries that provide answers to the questions FROM all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+/*Find all animals whose name ends in "mon".*/
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+/*List the name of all animals born BETWEEN 2016 AND 2019.*/
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+/*List the name of all animals that are neutered and have less than 3 escape attempts.*/
+SELECT * FROM animals WHERE neutered = true AND escape_attempts < 3;
+
+/*List the date of birth of all animals named either "Agumon" or "Pikachu".*/
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
+
+/*List name and escape attempts of animals that weigh more than 10.5kg*/
+SELECT name, escape_attempts FROM animals WHERE weight_kg >= 10.5;
```

Comment:
- You have done great so far :+1: But there are a few changes you need to make before moving to the next milestone. :+1: :+1: 
- [ ] This query says `List name and escape attempts of animals that weigh more than 10.5kg`, your current implementation includes even animals that weigh 10.5kg which is not correct, I'll suggest you remove the `=` here. :+1: :wave: